image_test.go

```diff
@@ -447,7 +447,7 @@ func TestExportImport(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to create output tar: %v", err)
 	}
-	err = rc.ImageExport(ctx, rIn1, fileOut1)
+	err = rc.ImageExport(ctx, []ref.Ref{rIn1}, fileOut1)
```

Comment:
These are indications of a breaking change when existing tests have to be rewritten for different parameter types.

Reply:
As I commented before, I did not expect this was an interface used outside this project, but only for access between packages in this project. I can easily change this and push a fixed commit.